docs/src/tutorials/Add-New-Network.md

```diff
@@ -0,0 +1,164 @@
+- [ ] [subkey](https://docs.substrate.io/v3/tools/subkey/#installation)
+- [ ] a clone of [parity-signer repository](https://github.com/paritytech/parity-signer). In the repository go to `generate_message` crate:
+
+```
+cd rust/generate_message
+cargo build
+cargo run restore_defaults
+```
+
+With `subkey` you need to generate a private and public keys (`<secret-phrase>` and `Public key (hex)`). You will be able to update a network only with metadata that is signed with the same keys as network specs that defined the network for your Signer instance.
+
+```
+subkey generate
+```
+
+---
+
+## Network Specs
+
+### Get Network Specs
+
+In `parity-signer/rust/generate_message`
+
+```
+cargo run add_spec -u <network-ws-url> -<crypto>
+
+```
+```
+// i.g.
+cargo run add_specs -u wss://statemint-rpc.polkadot.io -sr25519
+
+```
+
+For networks supporting several tokens:
+
+```
+cargo run add_spec -u <network-ws-url> -<crypto> -token <decimals> <SYMBOL>
+
+```
+```
+// i.g.
+cargo run add_spec -u wss://karura-rpc-0.aca-api.network -sr25519 -token 12 KAR
+
+```
+
+Now your `<specs-file>` is in `parity-signer/rust/files/for_signing`
+
+
+### Sign Network Spec
+
+#### Get signature
+
+In `parity-signer/rust/files/for_signing`
+
+```
+cat <spec-file> | subkey sign --suri <seed-phrase>
+```
+
+```
+// i.g.
+cat sign_me_add_specs_statemint_sr25519 | subkey sign --suri "fluid upon nuclear level middle funny suffer author group shrimp tornado relax"
+```
+
+This will return a `<signature>` you need to make a signed QR.
+
+#### Make signed QR
+
+In `parity-signer/rust/generate_message`
+
+```
+cargo run make -qr -crypto <crypto> -msgtype add_specs -payload <spec-file> -verifier -hex <signature>
+```
+
+```
+// i.g.
+cargo run make -qr -crypto sr25519 -msgtype add_specs -payload sign_me_add_specs_statemint_sr25519 -verifier -hex 0x927c307614dba6ec42f84411cc1e93c6579893859ce5a7ac3d8c2fb1649d1542 -signature -hex fa3ed5e1156d3d51349cd9bb4257387d8e32d49861c0952eaff1c2d982332e13afa8856bb6dfc684263aa3570499e067d4d78ea2dfa7a9b85e8ea273d3a81a86
+```
+
+Now your `<spec-qr>` is in `parity-signer/rust/files/signed`
+
+### Feed Network Specs into Signer
+
+In Signer open scanner, scan your your `<spec-qr>` and approve chain specs.
+
+
+## Network Metadata
+
+### Get Network Metadata
+
+In `parity-signer/rust/generate_message`
+
+```
+cargo run load_metadata -d -u `<network-ws-url>`
+```
+
+```
+// i.g.
+cargo run load_metadata -d -u wss://statemint-rpc.polkadot.io
+```
+
+This will fetch fresh `<metadata-file>`, update the database with it, and - most relevant to us currently - generate file with message body in `parity-signer/rust/files/for_signing`. 
+
+### Sign Network Metadata
+
+#### Get Signature
+
+In `parity-signer/rust/files/for_signing`
+
+```
+cat <metadata-file> | subkey sign --suri <seed-phrase>
+```
+
+```
+// i.g.
+cat sign_me_load_metadata_statemintV800 | subkey sign --suri "fluid upon nuclear level middle funny suffer author group shrimp tornado relax"
```

Comment:
Do we have a policy about posting random suri? I had an impression that it is nice to use default well-known-keys in examples unless it is essential to show other phrase. I'm not sure, please consider this and decide whether it is appropriate.